tests/testthat/test-sparsegl.R

```diff
@@ -1,8 +1,285 @@
+library(Matrix)
```

Comment:
You shouldn't need to load `Matrix` since it's a dependency of `sparsegl`.

The others need to be added to `Suggests:` in `DESCRIPTION`. Easiest is to call
```r
usethis::use_package("glmnet", type = "Suggests")
```